src/containers/my-cooperations/cooperation-completion/CooperationCompletion.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import SettingItem from '~/components/setting-item/SettingItem'
+import AppButton from '~/components/app-button/AppButton'
+import AppSelect from '~/components/app-select/AppSelect'
+
+import { cooperationAccessValues } from '~/containers/my-cooperations/cooperation-completion/CooperationCompletion.constants'
+import { styles } from '~/containers/my-cooperations/cooperation-completion/CooperationCompletion.styles'
+import {
+  ButtonVariantEnum,
+  SizeEnum,
+  CooperationMaterialsAccessEnum
+} from '~/types'
+
+const CooperationCompletion = () => {
+  const { t } = useTranslation()
+
+  const [materialsAccess, setMaterialsAccess] =
+    useState<CooperationMaterialsAccessEnum>(
+      CooperationMaterialsAccessEnum.OneMonthAccess
+    )
+
+  return (
+    <Box>
+      <Box sx={styles.divider}></Box>
```

Comment:
You can use `<Divider>` instead of `<Box>`

Reply:
changed)